webapp.js

```diff
@@ -318,8 +317,12 @@ function formatDailyInput(aggregatedDailyData, datesInput) {
 };
 
 function formatCumulativeInput(daily, remaining) {
-  var remainingCount = remaining[0] ? remaining[0].count : 0;
+  var totalCount = remaining[0] ? remaining[0].count : 0;
   var countsCopy = daily.slice();
+  var dailyCount = countsCopy.reduce(function(acc, item) {
```

Comment:
Can just call `reduce` directly on `daily` since it doesn't mutate the original.